conda-store-server/environment.yaml

```
@@ -18,6 +18,7 @@ dependencies:
   - redis-py
   - sqlalchemy<=1.4.47
   - alembic
+  - platformdirs
```

I wonder if we should pin this to at least current major. The library has [received some breaking updates in the past](https://platformdirs.readthedocs.io/en/latest/changelog.html) that would have affected our configuration.

```suggestion
  - platformdirs >=4.0,<5.0a0
```

We can then unpin as needed if there are major releases that do not break.